Data_Structures_Answers.md

```diff
@@ -1,18 +1,25 @@
 Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
+    O(n) time complexity
 
 2. What is the space complexity of your ring buffer's `append` function?
+    O(1) space complexit after its full or O(n) before its full.  
```

Comment:
It is O(1) but the reason is because python always keeps track of how many elements are in a list so we arent allocating any additional memory to append. HOWEVER you can implement it in a more inefficient way and then that WOULD be O(n)